pkg/network/node/ovscontroller_test.go

```diff
@@ -1185,14 +1185,14 @@ var expectedFlows = []string{
 	" cookie=0, table=100, priority=1, reg0=42, ip, nw_dst=192.168.1.1/32, actions=goto_table:101",
 	" cookie=0, table=100, priority=0, actions=goto_table:101",
 	" cookie=0, table=101, priority=150, ct_state=+rpl, actions=output:2",
-	" cookie=0, table=101, priority=100, ip, reg0=37, actions=group:37",
+	" cookie=0, table=101, priority=100, ip, reg0=37, actions=ct(commit),group:37",
```

Comment:
hm... `TestRuleVersion` isn't really doing a good job of testing egress IPs... we should fix that (have actual proper rules for both a local egress IP and a remote one)